.github/workflows/tester.yml

```
@@ -0,0 +1,34 @@
+name: "Tester"
```

```suggestion
name: "Tests"
```

lets rename the file as well to `tests.yml`